<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BkhDetail</class>
 <widget class="QDialog" name="BkhDetail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>36</y>
     <width>469</width>
     <height>415</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="adc">
    <attribute name="title">
     <string>ADC/DAC</string>
    </attribute>
    <widget class="QEForm" name="qeform_13">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>14</y>
       <width>441</width>
       <height>314</height>
      </rect>
     </property>
     <property name="uiFile" stdset="0">
      <string>../ui-common/adcdac.ui</string>
     </property>
     <property name="variableSubstitutions" stdset="0">
      <string>P=CHA,R=3132,T=4132,N=2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>342</y>
       <width>99</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>ADC Conversion</string>
     </property>
    </widget>
    <widget class="QPushButton" name="coefADCPB">
     <property name="geometry">
      <rect>
       <x>362</x>
       <y>342</y>
       <width>83</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Coefficients</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="hidden">
    <attribute name="title">
     <string>Hidden</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget_2">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>14</y>
       <width>409</width>
       <height>314</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Mot 1</string>
      </attribute>
      <widget class="QEForm" name="qeform_14">
       <property name="geometry">
        <rect>
         <x>8</x>
         <y>8</y>
         <width>378</width>
         <height>260</height>
        </rect>
       </property>
       <property name="uiFile" stdset="0">
        <string>../ui-common/hiddenreg.ui</string>
       </property>
       <property name="variableSubstitutions" stdset="0">
        <string>Q=2531-1,N=2</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Mot 2</string>
      </attribute>
      <widget class="QEForm" name="qeform_15">
       <property name="geometry">
        <rect>
         <x>8</x>
         <y>8</y>
         <width>378</width>
         <height>260</height>
        </rect>
       </property>
       <property name="uiFile" stdset="0">
        <string>../ui-common/hiddenreg.ui</string>
       </property>
       <property name="variableSubstitutions" stdset="0">
        <string>Q=2531-2,N=2</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>ADC</string>
      </attribute>
      <widget class="QEForm" name="qeform_16">
       <property name="geometry">
        <rect>
         <x>8</x>
         <y>8</y>
         <width>378</width>
         <height>260</height>
        </rect>
       </property>
       <property name="uiFile" stdset="0">
        <string>../ui-common/hiddenreg.ui</string>
       </property>
       <property name="variableSubstitutions" stdset="0">
        <string>Q=3132,N=2</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>DAC</string>
      </attribute>
      <widget class="QEForm" name="qeform_17">
       <property name="geometry">
        <rect>
         <x>8</x>
         <y>8</y>
         <width>378</width>
         <height>260</height>
        </rect>
       </property>
       <property name="uiFile" stdset="0">
        <string>../ui-common/hiddenreg.ui</string>
       </property>
       <property name="variableSubstitutions" stdset="0">
        <string>Q=4132,N=2</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="bk9000">
    <attribute name="title">
     <string>BK9000</string>
    </attribute>
    <widget class="QEForm" name="qeFormBK9">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>8</y>
       <width>403</width>
       <height>340</height>
      </rect>
     </property>
     <property name="uiFile" stdset="0">
      <string>../ui-common/bk9000.ui</string>
     </property>
     <property name="variableSubstitutions" stdset="0">
      <string>P=CHA,N=2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="debugPB">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>354</y>
       <width>72</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Debug</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="closePB">
   <property name="geometry">
    <rect>
     <x>414</x>
     <y>6</y>
     <width>56</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QEForm</class>
   <extends>QWidget</extends>
   <header>QEForm.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
